examples/getting-started-app/README.md

```diff
@@ -84,139 +92,14 @@ You also need to install `vagrant-hosts`, `vagrant-vbguest` and `vagrant-scp` pl
-[vagrant@vm1 ~]$ exit
-$ vagrant scp vm1:~/getting-started-app-1.0.0-0.rpm .
-```
-
-The last command copies RPM from VM on your local machine.
-
-Now, you can stop the VM:
-
-```bash
-$ vagrant halt vm1
-```
-
-Now, we have the `getting-started-app-1.0.0-0.rpm` file in the current directory.
-The next step is to deploy this package on servers and set up a cluster.
-
-## What does Tarantool Cartridge role do?
-
-Here follows a description of deploy steps, just to understand the way
-Tarantool Cartridge applications should be deployed.
-You can skip this section if you are not interested in understanding how it works
-under the hood, and proceed directly to [deployment steps](#deploying-with-ansible).
-
-### Deploying the package
-
-The first step is to install the application package on a deployment server.
-Here we will create a user `tarantool` with a group `tarantool` and some directories
-for our app:
-
-* `/etc/tarantool/conf.d/` - directory for instances configuration;
-* `/var/lib/tarantool/` - directory to store instances snapshots;
-* `/var/run/tarantool/` - directory to store PID-files and console sockets.
-
-Application code will be placed in the `/usr/share/tarantool/${app-name}` directory.
-If you use Tarantool Enterprise, `tarantool` and `tarantoolctl` binaries will be
-delivered with the package and placed there too.
-Otherwise, your RPM package has a Tarantool dependency and `yum` (RPM packages
-manager) will install open-source Tarantool.
-
-The package also contains `/etc/systemd/system/${app-name}.service` and
-`/etc/systemd/system/{app-name}@.service`
-[systemd unit files](https://www.digitalocean.com/community/tutorials/understanding-systemd-units-and-unit-files).
-
-### Starting instances
-
-When you call `systemctl start getting-started-app@storage-1`, systemd starts
-the `storage-1` instance of the `getting-started-app` service
-(see [systemd template units](https://fedoramagazine.org/systemd-template-unit-files/)).
-This instance will look up its configuration across all
-sections of the YAML file(s) stored in `/etc/tarantool/conf.d/*`.
-
-See [documentation](https://www.tarantool.io/en/doc/2.2/book/cartridge/cartridge_dev/#configuring-instances)
-for details on instance configuration.
-
-### Setting up the topology
-
-After the instances are started, you need to set up the cluster topology,
-manage authorization parameters, browse application configuration,
-enable automatic failover, and bootstrap vshard.
-
-All these actions can be performed using the cluster WebUI or the `cartridge`
-module API.
-
-To use the API, you can connect to the instance console using a socket:
+## Deploying with Ansible
 
-```bash
-$ tarantoolctl connect /var/run/tarantool/getting-started-app.storage-1.control
-tarantool> require('cartridge').is_healthy()
-```
+Now, we will start 3 instances on 2 different servers, join them in 2 replica
+sets, bootstrap vshard, enable automatic failover, set up authorization and
```

Comment:
> failover

I think it should be noticed that current implementation supports only "eventual" failover mode. At least two words in "Manage failover" section

Reply:
Added a note to this README and the main README.